recipe/meta.yaml

```diff
@@ -45,23 +62,91 @@ requirements:
     - cytoolz >=0.7.3
     # cloudpickle is necessary to provide the 'processes' scheduler for dask
     - cloudpickle >=0.2.1
+    - _openmp_mutex  # [linux]
+    - tifffile >=2019.7.26
+    # scikit-image depends on dask-array
+    # which requires numpy, dask-core and toolz (cytoolz for speed)
+    - dask-core >=1.0.0,!=2.17.0
```

Comment:
dask-core and toolz are already mentioned above. It's a duplication

Reply:
thanks I will remove it.  I just found an issue on py310 on Windows for tifffile.c header, which tries to use undeclared type 'ssize_t'.  I am testing a fix for it